examples/Google.Cloud.Functions.Examples.TimeZoneConverter/README.md

```diff
@@ -0,0 +1,60 @@
+# TimeZoneConverter example function
+
+This function uses the [Noda Time](https://nodatime.org) library to
+perform time zone conversions based on the [IANA time zone
+database](https://www.iana.org/time-zones).
+
+## Parameters
+
+All input is provided via the following query parameters:
+
+- `fromZone`: The ID of the time zone to convert from
+- `toZone`: The ID of the time zone to convert to
+- `when`: The local date/time to convert, from `fromZone` to `toZone`
```

Comment:
Local here means fromZone, right? I would say so explicitly, here and everywhere.

Reply:
It's slightly tricky. The value itself isn't in any zone; it's *interpreted* with respect to `fromZone`. Will have a think about another way of expressing that.